### PR TITLE
sci-mathematics/netgen: add 6.2.2204

### PR DESCRIPTION
**Pull request by waebbl**
- OpenMP doesn't seem to be used, remove USE flag
- dev-tcltk/tix doesn't seem to be used, remove dependency
- dev-tcltk/togl is build internal and doesn't use a system togl installation
- Stub generation from pybind11 doesn't currently work in a sandbox, see https://github.com/NGSolve/netgen/issues/132
- only depend on sci-libs/metis for USE=mpi

Signed-off-by: Bernd Waibel <waebbl-gentoo@posteo.net>

### CODE REVIEW

**Comment by waebbl** [comment]
How about moving this package to different category, like media-gfx? IMO it's not really a maths package. Also the desktop file we provide gives it a category of Graphics, instead of Education, Science, Maths.

**Comment by waebbl** [comment]
Actually, the libtogl.a library is referenced in the netgen-targets.cmake file. So we have to leave it for now.

File: sci-mathematics/netgen/netgen-6.2.2204.ebuild [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+	fi
+	cmake_src_configure
+}
+
+src_install() {
+	cmake_src_install
+	use python && python_optimize
+
+	local NETGENDIR="/usr/share/${PN}"
+	echo -e "NETGENDIR=${NETGENDIR}" > ./99netgen
```
Comment (AndrewAmmerlaan) [inline sci-mathematics/netgen/netgen-6.2.2204.ebuild:125]: `|| die`

Comment (waebbl) [reply sci-mathematics/netgen/netgen-6.2.2204.ebuild:125]: Didn't change this line ;) but yeah, you're right.

**Comment by waebbl** [comment]
> Yeah I think we should move it.

Then I move it to media-gfx on my next push with your permission.

**Comment by AndrewAmmerlaan** [comment]
Thanks :+1:

**Comment by waebbl** [comment]
Thanks for merging @AndrewAmmerlaan 

I'd like to discuss the possibility of getting the package into ::gentoo. In my overlay, I tested the support in FreeCAD for this version and it builds fine. I'd be happy, if we could add this support to the FreeCAD versions in tree as well.

WDYT?

**Comment by AndrewAmmerlaan** [comment]
> Thanks for merging @AndrewAmmerlaan
> 
> I'd like to discuss the possibility of getting the package into ::gentoo. In my overlay, I tested the support in FreeCAD for this version and it builds fine. I'd be happy, if we could add this support to the FreeCAD versions in tree as well.
> 
> WDYT?

Yeah since this is an optional dependency of freecad I think it is a good candidate for moving to ::gentoo. The ebuild looks fine, but there is a test failure with `USE="python"` that we should fix. It looks like the failure occurs because the package is not yet installed (it can't find its own modules), setting an appropriate `PYTHONPATH` should fix it.

**Comment by waebbl** [comment]
> > Thanks for merging @AndrewAmmerlaan
> > I'd like to discuss the possibility of getting the package into ::gentoo. In my overlay, I tested the support in FreeCAD for this version and it builds fine. I'd be happy, if we could add this support to the FreeCAD versions in tree as well.
> > WDYT?
> 
> Yeah since this is an optional dependency of freecad I think it is a good candidate for moving to ::gentoo. The ebuild looks fine, but there is a test failure with `USE="python"` that we should fix. It looks like the failure occurs because the package is not yet installed (it can't find its own modules), setting an appropriate `PYTHONPATH` should fix it.

I look into the test failure. I also want to polish the ebuild some more and check adding support for `dev-libs/spdlog`

**Pull request closed without merging**